server/pulp/server/managers/content/orphan.py

```diff
@@ -224,10 +224,16 @@ def delete_orphans_by_type(content_type_id, content_unit_ids=None):
         """
 
         content_units_collection = content_types_db.type_units_collection(content_type_id)
+        content_model = plugin_api.get_unit_model_by_id(content_type_id)
 
+        try:
+            unit_key_fields = units_controller.get_unit_key_fields_for_type(content_type_id)
+        except ValueError:
+            raise MissingResource(content_type_id=content_type_id)
+
+        fields = ('_id', '_storage_path') + unit_key_fields
         count = 0
```

Comment:
Is this needed? Looks like ``count`` doesn't get used until below where it gets again set to ``0``.